gamco.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  spec.add_dependency "rails", ">= 5.0"
```

Comment:
isn;t rails 5 too low?

Reply:
I can require a highest version. I was trying to support most of our apps and I do not know if all our apps are updated.

Reply:
ok, there is still one app on out side in process toupdate to rails 6, so yeah probably leave it at 5 for now